runtimevar/etcdvar/etcdvar.go

```diff
@@ -130,3 +143,9 @@ func (w *watcher) WatchVariable(ctx context.Context, prev driver.State) (driver.
 func (w *watcher) Close() error {
 	return nil
 }
+
+// WatchOptions allows the specification of various options to a watcher.
+type WatchOptions struct {
+	// WaitTime controls the frequency of retries after an error. Defaults to 30 seconds.
```

Comment:
nit: "...Default to defaultWait"?

Reply:
defaultWait is a private var that's not visible in the godoc.
For example, see the existing godoc [here](https://godoc.org/github.com/google/go-cloud/runtimevar/runtimeconfigurator), the docstring references `defaultWait` but you can't see it anywhere.